app/emails/templates/user_existing_application.md

```diff
@@ -0,0 +1,10 @@
+Thanks for requesting access to the Digital Marketplace.
+
+It looks like an application for {{business_name}} has already been started by {{representative}}. Once the application has been approved, they can [add you as a contributor](https://marketplace.service.gov.au/sellers-guide).
```

Comment:
You can add a #contributor anchor to the link after https://github.com/AusDTO/dto-digitalmarketplace-buyer-frontend/pull/566